### ISSUE TITLE
Assertions should work against a list of ElementHandles

### ISSUE THREAD

**Issue by ramraphael** [playwright-community/expect-playwright#28]
Hi here,

Currently, all assertions are designed to check a single element. It would be helpful to check multiple elements (list items, table cells) and assert that they contain certain text.

Just spitballing here, but the `expect(page).toHaveText(selector, text, options)` selector would need some way to allow users to indicate that the selector should select multiple elements, each of which will have its inner text checked.

The `expect(elementHandle).toHaveText(text, options)` should be able to detect if it is passed a single elementHandle or an array of elementHandles.

Would love thoughts from the community. Happy to collaborate! =D

**Comment by creage** [comment]
@mmarkelov can we have this feature implemented please?

**Comment by mmarkelov** [comment]
@creage yeah! I suppose so. I just need to think about the way of this matcher.

**Comment by creage** [comment]
How about new `all` chainable?

Something like `expect(page).all.toHaveText(selector, text, options)`?

**Comment by mmarkelov** [comment]
@creage IDK if it's possible to chain it in this way, but this looks good to me.
